### ISSUE TITLE
Adding new corporations

### ISSUE THREAD

**Issue by FishyStack-source** [muffix/couleurbummel#111]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### This is not a data update request

- [X] I am not requesting a data update

### Describe the solution you'd like

Hi,
I've been trying to add two new corporations to the app for a while now, but I can't find anyone to contact.
If anyone know who to contact, or has a contact of someone who can help me, please send through mail: korp.vytis@gmail.com
Thanks in advance

**Comment by memnonic** [comment]
Up, same issue. If needed please contact at kontakt@konwentpolonia.pl.

**Comment by FishyStack-source** [comment]
Hello,

It is great that you contacted me.

I heard that Konvent polonia is organising Baltic Nations kommers. I was
trying to find an email to contact the organizers, could you help me with
that?

And about Couleurbummel. I managed to find an admin/founder email, and I am
happy to share it with you.
Björn Marschollek ***@***.*** takes his time to add new
fraternities, but it's progress.

Hope it helps!

sincerely,
president of Korp!Vytis
Senj.Emilis Varžinskas

AD MELIOREM!

2025-01-15, tr, 05:25 Maciej Pior ***@***.***> rašė:

**Comment by memnonic** [comment]
> I heard that Konvent polonia is organising Baltic Nations kommers. I was
> trying to find an email to contact the organizers, could you help me with
> that?

Yep, that's right. Just grab the main contact email from our website, it's checked regularly by our secretary.
 
> And about Couleurbummel. I managed to find an admin/founder email, and I am
> happy to share it with you.
> Björn Marschollek ***@***.*** takes his time to add new
> fraternities, but it's progress.

Thanks for your help, but unfortunately github seems to have obfuscated the email address. 😁 

fil. Maciej Piór

**Comment by FishyStack-source** [comment]
I sent email to your fraternity email, maybe the email is alright there🤔
